storybook/src/Storybook/SingleItemProxyModel.qml

```diff
@@ -0,0 +1,10 @@
+import SortFilterProxyModel 0.2
```

Comment:
Makes sense to move it to models folder?

Reply:
I would leave it for later :)